src/target/arc_ocd.c

```diff
@@ -141,6 +141,13 @@ int arc_ocd_deassert_reset(struct target *target)
 	/* deassert reset lines */
 	jtag_add_reset(0, 0);
 
+	if (!target->reset_halt && target->state == TARGET_RESET)
+	{
+		CHECK_RETVAL(target_halt(target));
+		// Resume current target handling breakpoints.
```

Comment:
Last two word look very confusing. Do you imply that following call to **target_resume()** will write all breakpoints already set by the user? Try to restructure sentence.